components/tikv_util/src/buffer_vec.rs

```diff
@@ -598,8 +634,18 @@ mod tests {
         assert_eq!(v3.total_len(), 3);
         assert_eq!(format!("{:?}", v3), "[0C00, null, null, AA]");
 
+        v3.extend(&[0xAA, 0xAB, 0xCC]);
+        assert_eq!(v3.len(), 5);
+        assert_eq!(v3.total_len(), 6);
+        assert_eq!(format!("{:?}", v3), "[0C00, null, null, AA, AAABCC]");
+
+        v3.extend(&[]);
+        assert_eq!(v3.len(), 6);
+        assert_eq!(v3.total_len(), 6);
+        assert_eq!(format!("{:?}", v3), "[0C00, null, null, AA, AAABCC, null]");
+
```

Comment:
I found there are implemention for `Extend<&'a u8>`, is it proper to adding testing like:
```
v3.extend(vec![0xAA]);
...
```
?